modules/enabling_encapsulation.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/optimizing-cpu-usage.adoc
+
+[id="enabling-encapsulation_{context}"]
+= Working with mount namespace encapsulation
+
+As an OpenShift system administrator, you want `systemd` to consume less resources so that you can run more workloads on my system. You want to enable and disable this setting, and verify whether encapsulation is in effect.
+
+== Enabling and disabling encapsulation
+
+.Prerequisites
+
+* You have access to the cluster as a user with the cluster-admin role.
+
+* You have installed the OpenShift CLI (oc).
+
+* You have created the `MachineConfig` object.
+
+.Procedure
+
+. Add the following YAML to the `MachineConfig.yaml` file to control whether encapsulation is enabled or disabled:
+
++
+[source,yaml]
+----
+  apiVersion: machineconfiguration.openshift.io/v1
+  kind: MachineConfig
+  metadata:
+    labels:
+      machineconfiguration.openshift.io/role: master
+    name: 99-kubens-master
+  spec:
+    config:
+      ignition:
+        version: 2.2.0
+      systemd:
+        units:
+        - enabled: true <1>
+          name: kubens.service
+
+  apiVersion: machineconfiguration.openshift.io/v1
+  kind: MachineConfig
+  metadata:
+    labels:
+      machineconfiguration.openshift.io/role: worker
+    name: 99-kubens-worker
+  spec:
+    config:
+      ignition:
+        version: 2.2.0
+      systemd:
+        units:
+        - enabled: true <2>
+          name: kubens.service
+
+----
+<1> If `enabled` is set to true, encapsulation is in effect for the primary node. Both <1> and <2> need to be set to true. If either are false, encapsulation will not be enabled. If you wish to disable encapsulation, set both <1> and <2> to `false`.
```

Comment:
Primary node?
This line is confusing, why not just say this needs to be set to true to enable the feature on master nodes

Reply:
Writers have to follow inclusive language guidelines, including the use of master/slave terminology.

Reply:
My understanding of the inclusive language guidlines is that if there is pre-existing technical terminology it's okay to keep referring to it.  Until we actual label the nodes differently in the OpenShift product, isn't it confusing to talk about "primary nodes" when these are represented in the product as "node with MCP role of `master`"?

Reply:
But if we're consistently referring to "nodes labeled with `master`" as "primary nodes" throughout the documentation, here's my suggested edit: 

```suggestion
<1> If `enabled` is set to true, encapsulation is in effect for all primary nodes.
```

Reply:
Looking at the OpenShift documentation, the term *master* node is consistently used.

While I agree with the benefit of  using inclusive language, the change needs to be done consistently, both in the product as well as in the documentation

Reply:
"Control plane node" is the preffered usage, see: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#control-plane

Reply:
In this case "master" is in the code. We can socialize the new usage, along with a nod to the code:

If `enabled` is set to true, encapsulation is in effect for all control plane (previously known as "master") nodes.